src/dex/algebra/algebra.ts

```diff
@@ -165,7 +165,7 @@ export class Algebra extends SimpleExchange implements IDex<AlgebraData> {
     token0,
     token1,
   }) => {
-    const logPrefix = '[Algebra.onPoolCreatedDeleteFromNonExistingSet]';
+    const logPrefix = `[${this.dexKey}.onPoolCreatedDeleteFromNonExistingSet]`;
```

Comment:
`dexKey` is redundant, all logs are already namespaced with `dexKey` and `chainId`

Reply:
removed